bin/src/utils.rs

```diff
@@ -6,6 +6,15 @@ use crate::menu;
 
 pub(crate) async fn main_window_show(app: &AppHandle) {
     if let Some(w) = app.get_webview_window("main") {
+        let windows = app.webview_windows();
+
+        if !windows.is_empty() {
+            for window in windows.values() {
+                window.set_focus().unwrap();
+            }
+        }
+
+        w.set_focus().unwrap();
```

Comment:
this code works, but I don't think it belongs in `main_window_show` at all
how about creating a separate function for this case where we want to refocus all windows (and open main if it's not yet)